internal/transport/http2_client.go

```diff
@@ -1006,24 +1008,29 @@ func (t *http2Client) Close(err error) {
 		t.kpDormancyCond.Signal()
 	}
 	t.mu.Unlock()
-	// Per HTTP/2 spec, a GOAWAY frame must be sent before closing the
-	// connection. See https://httpwg.org/specs/rfc7540.html#GOAWAY.
+	var st *status.Status
+	// Per HTTP/2 spec, a GOAWAY frame must be sent before closing the connection.
```

Comment:
Please ensure that new comment lines are wrapped at 80-cols. See: https://google.github.io/styleguide/go/decisions#comment-line-length